src/axom/spin/OctreeLevel.hpp

```diff
@@ -158,7 +158,7 @@ class OctreeLevel
    */
   CoordType maxCoord() const
   {
-    return (1<< m_level) -1;
+    return (CoordType(1)<< m_level) -CoordType(1);
```

Comment:
This one (here and elsewhere) was a bit dubious.

For 64-bit shift operations on numeric literals, we need to ensure that we are using a 64-bit register by casting to the appropriate int type.